components/brave_rewards/content/android_util.cc

```diff
@@ -1,17 +0,0 @@
-/* Copyright (c) 2019 The Brave Authors. All rights reserved.
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at https://mozilla.org/MPL/2.0/. */
-
-#include "brave/components/brave_rewards/content/android_util.h"
```

Comment:
Do we need to delete the header file that corresponds with this as well?

Reply:
Yep - good catch